config/jobs/kubernetes/wg-k8s-infra/wg-k8s-infra-canaries.yaml

```diff
@@ -0,0 +1,366 @@
+periodics:
+- cron: '1 12 * * *' # Run daily at 4:01PST (12:01 UTC)
+  name: ci-kubernetes-e2e-gce-scale-correctness-canary
+  cluster: k8s-infra-prow-build
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-scalability-common: "true"
+    preset-e2e-scalability-periodics: "true"
+    preset-e2e-scalability-periodics-master: "true"
+  decorate: true
+  decoration_config:
+    timeout: 270m
+  annotations:
+    testgrid-dashboards: wg-k8s-infra-canaries
+    testgrid-tab-name: gce-master-scale-correctness-canary
+    description: "Uses kubetest to run correctness tests against a 5000-node cluster created with cluster/kube-up.sh"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210601-ea6aa4e-master
+      command:
+      - runner.sh
+      - /workspace/scenarios/kubernetes_e2e.py
+      args:
+      - --cluster=gce-scale-cluster
+      - --env=CONCURRENT_SERVICE_SYNCS=5
+      - --env=HEAPSTER_MACHINE_TYPE=e2-standard-32
+      - --extract=ci/latest-fast
+      - --extract-ci-bucket=k8s-release-dev
+      # Overrides CONTROLLER_MANAGER_TEST_ARGS from preset-e2e-scalability-periodics.
+      - --env=CONTROLLER_MANAGER_TEST_ARGS=--profiling --kube-api-qps=100 --kube-api-burst=100 --endpointslice-updates-batch-period=500ms
```

Comment:
Value of this flag has changed in https://github.com/kubernetes/test-infra/pull/22446

Reply:
Fixed.